app/experimenter/docs/adrs/0001-create-new-nimbus-application.md

```diff
@@ -0,0 +1,119 @@
+# Create A New Web Application To Manage Nimbus Experiments
+
+* Status: proposed
+* Deciders: Kate Hudson, Jared Lockhart, TBD
+* Date: 2020-08-18
+
+## Context and Problem Statement
+
+Experimenter is a web application developed solely for the purpose of managing product experiments at Mozilla.  It was originally designed to manage the Shield experimentation workflow, and has expanded to include a comprehensive workflow for managing Normandy experiments.  Project Nimbus is being developed with very different core assumptions than the previous experimentation platforms, and it is unclear whether building on top of the existing Experimenter infrastructure or creating a new purpose specific application will best suit the project's needs.
+
+## Decision Drivers <!-- optional -->
+
+* Meeting Project Nimbus' needs in a rapid, continuous fashion with minimal downtime
+* Contributability by a large, distributed group of people with different backgrounds (web engineers, data engineers, data scientists)
+* Fitting cohesively into the FXA engineering pipeline/culture
+* Continuing to support the existing experimentation (Normandy) workflow
+
+## Considered Options
+
+* Build On Top Of Experimenter
+* Start A New Application In Python/Django
+* Start A New Application in TypeScript/Node
+
+## Decision Outcome
+
+Chosen option: TBD
+
+### Positive Consequences
+
+* [e.g., improvement of quality attribute satisfaction, follow-up decisions required, …]
+* …
+
+### Negative Consequences
+
+* [e.g., compromising quality attribute, follow-up decisions required, …]
+* …
+
+## Pros and Cons of the Options
+
+### Build On Top Of Experimenter
+
+Much of the existing Experimenter infrastructure can be reused, which gives Project Nimbus a well established framework to build on top of.
+
+- Pros
+  - Single source of truth for all past/future experiments
+    - Easily discover past results
+    - Correlate past/future experiments easily
+    - Generate high level reports for leadership/stakeholders across all experiment types
+    - Consistent APIs for dependent automated tasks (ie data engineering/science)
+  - Reuse existing components
+    - Database schemas are largely reusable
+    - Public APIs work seamlessly for Normandy and Nimbus experiments
+    - External service integrations (bugzilla/remote settings/email/auth0) work seamlessly
+  - Comprehensive development/testing/deployment infrastructure in place
+    - 100% unit testing coverage across Python/JS/TS
+    - Automated Selenium Integration tests for frontend
+    - Continuous Deployment
+- Cons
+  - Existing workflow/UI not applicable
```

Comment:
I haven't seen any substantial context on Nimbus. I'm also not familiar with how Experimenter works for Shield and Normandy. Would be useful to see what the current UI is and how/why it is not applicable to Nimbus.

Reply:
I commented above with some more detailed description of those differences, I could move it into the ADR itself if it would be helpful?

Reply:
We should add a pro of "dynamic risk mitigation".  I embrace designing for the simplest case - and agree Nimbus is removing a lot of the risk mitigation steps needed.  But in the design -there have been several cases where risk will need to set on different path for mitigation (partner involved, new metrics needed, new targeting needed).  The goal is 100% to move to lighter weight/no review whenever possible - but kick the ones that need more hands-on into that flow.